src/main/java/com/checkout/AbstractCheckoutSdkBuilder.java

```diff
@@ -21,11 +19,19 @@
         return this;
     }
 
+    /**
+     * @deprecated in favor to use Environment
+     */
+    @Deprecated
     public AbstractCheckoutSdkBuilder<T> uri(final URI uri) {
         this.uri = uri;
         return this;
     }
 
+    /**
+     * @deprecated in favor to use Environment
+     */
+    @Deprecated
```

Comment:
Can you please provide a more informative message here? I was thinking of something like: "In a future release, Files API will be enabled by default and it will use the same environment set in #environment()"